### PR TITLE
Add new API data from old browser compat tables

### PR DESCRIPTION
**Pull request by vinyldarkscratch**
Based upon [this gist](https://gist.github.com/Elchi3/b6863aa227fc4cd51a247957380eca5e), this PR continues some of last year's goal, adding in the last API tables.  Some of the copied data was slightly altered to accommodate for Chromium-based browsers.

### CODE REVIEW

**Comment by ddbeck** [comment]
Sorry, @vinyldarkscratch there were some other Element PRs before this one. Would you mind rebasing this? Thanks!

**Comment by vinyldarkscratch** [comment]
Done!

File: api/Document.json [hunk 1]
```diff
@@ -429,20 +429,17 @@
...
-              "version_added": null
+              "version_added": false
             },
             "safari": {
               "version_added": null
             },
             "safari_ios": {
               "version_added": null
             },
-            "samsunginternet_android": {
```
Comment (ddbeck) [inline api/Document.json]: I think these lines can stay

File: api/WebSockets.json [hunk 2]
```diff
@@ -0,0 +1,292 @@
+{
```
Comment (ddbeck) [inline api/WebSockets.json]: I think some of this data belongs in `WebSocket` (it looks as if there's no plural interface). That said, I think the original table contains some questionable data that'd be worth digging into a little more. Perhaps the right thing to do here is to drop this data from the PR entirely and start a separate PR for WebSocket data? That way we don't hold up the rest, which looks pretty good

Comment (vinyldarkscratch) [reply api/WebSockets.json]: Sure thing!  I'll separate it into a separate PR.  :wink:

**Pull request merged**